fabric-entity-events-v1/src/testmodClient/java/net/fabricmc/fabric/test/entity/event/client/EntityEventTestsClient.java

```diff
@@ -16,17 +16,34 @@
 
 package net.fabricmc.fabric.test.entity.event.client;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.Items;
 
 import net.fabricmc.api.ClientModInitializer;
 import net.fabricmc.fabric.api.client.rendering.v1.LivingEntityFeatureRenderEvents;
+import net.fabricmc.fabric.api.entity.event.client.ClientPlayerEvents;
 import net.fabricmc.fabric.test.entity.event.EntityEventTests;
 
 public class EntityEventTestsClient implements ClientModInitializer {
+	private static final Logger LOGGER = LoggerFactory.getLogger(EntityEventTestsClient.class);
+
 	@Override
 	public void onInitializeClient() {
 		LivingEntityFeatureRenderEvents.ALLOW_CAPE_RENDER.register(player -> {
 			return !player.getEquippedStack(EquipmentSlot.CHEST).isOf(EntityEventTests.DIAMOND_ELYTRA);
 		});
+
+		ClientPlayerEvents.MODIFY_PLAYER_MOVEMENT_DURING_USINGITEM.register(player -> {
+			LOGGER.info("Player {} is moving during using item.", player);
+
+			if (player.getMainHandStack().isOf(Items.BOW) && player.getEquippedStack(EquipmentSlot.FEET).isOf(Items.DIAMOND_BOOTS)) {
+				LOGGER.info("Player {} can move without slowdown becase of diamond boots on feet.", player);
+				player.input.movementForward *= 5F;
+				player.input.movementSideways *= 5F;
```

Comment:
So this PR, if you want to prevent slowdown, you multiply the move speed by 5? Uhhhhhhhhh

Have you considered what happens if two mods’s condition passes for trying to turn off the move speed? The two multiplying will end up causing the player to move 5 times as fast.

This PR is not suitable for multiple mods trying to turn off item use slowdown. It will need a rethinking

Reply:
Got it, I will try as soon as possible. Thank you!

Reply:
I changed the event to DISABLE_USINGITEM_SLOWDOWN. When the return value is true, the player can move without being slowed down, which will also resolve the issue of multiple mods affecting this simultaneously.